website/docs/guides/best-practices/how-we-style/1-how-we-style-our-dbt-models.md

```diff
@@ -0,0 +1,68 @@
+---
+title: How we style our dbt models
+id: 1-how-we-style-our-dbt-models
+---
+
+## Fields and model names
+
+- 👥 Models should be pluralized, e.g. `customers`, `orders`, `products`.
+- 🔑 Each model should have a primary key.
+- 🔑 The primary key of a model should be named `<object>_id`, e.g. `account_id` – this makes it easier to know what `id` is being referenced in downstream joined models.
+- 🔑 Keys should be string data types.
+- 🔑 Consistency is key! Use the same field names across models where possible, e.g. a key to the `customers` table should be named `customer_id` rather than `user_id` or 'id'.
+- ❌ Do not use abbreviations or aliases, emphasize readability over brevity, e.g. `cust` for `customer` or `o` for `orders`.
+- ❌ Avoid reserved words as column names.
+- ➕ Booleans should be prefixed with `is_` or `has_`.
+- 🕰️ Timestamp columns should be named `<event>_at`, e.g. `created_at`, and should be in UTC. If a different timezone is being used, this should be indicated with a suffix, e.g `created_at_pt`.
+- 📆 Dates should be named `<event>_date`, e.g. `created_date`.
+- 🔙 Events dates and times should be past tense, e.g. `created`, `updated`, `deleted`.
+- 💱 Price/revenue fields should be in decimal currency (e.g. `19.99` for $19.99; many app databases store prices as integers in cents). If non-decimal currency is used, indicate this with suffix, e.g. `price_in_cents`.
+- 🐍 Schema, table and column names should be in `snake_case`.
+- 🏦 Use names based on the _business_ terminology, rather than the source terminology. For example, if the source database uses `user_id` but the business calls them `customer_id`, use `customer_id` in the model.
```

Comment:
All the way through, or only as the models get closer to being marts? I saw a cool slide from a meetup which I don't know is for public consumption but I will DM you

Reply:
i let this marinate overnight, at first i was like 'let's add some nuance to this and soften it' but honestly, the more i thought about it the more i was like 'you're going to get a *lot* more clarity out of using unified terminology and it feels like a core function of AE to define and implement that' - i can't think of a compelling case for not immediately shifting to unified business terms at the staging layer except for exceptions where you have multiple source systems that need to be unified and have an underlying unification layer.